packages/mobile/src/screens/profile-screen/TracksTab.tsx

```diff
@@ -52,7 +50,6 @@ export const TracksTab = () => {
       leadingElementId={artist_pick_track_id}
       actions={tracksActions}
       lineup={lineup}
-      limit={track_count}
```

Comment:
if we don't include this, it still works properly because the lineup component can calculate that there are no more tracks to load once we reach the end.